rest/route/middleware.go

```diff
@@ -802,10 +801,12 @@ func AddCORSHeaders(allowedOrigins []string, next http.HandlerFunc) http.Handler
 		requester := r.Header.Get("Origin")
 		grip.Debug(message.Fields{
 			"op":              "addCORSHeaders",
+			"request":         r,
 			"requester":       requester,
 			"allowed_origins": allowedOrigins,
-			"adding_headers":  utility.StringSliceContains(allowedOrigins, requester),
+			"adding_headers":  util.StringContainsSliceRegex(allowedOrigins, requester),
 			"settings_is_nil": evergreen.GetEnvironment().Settings() == nil,
+			"headers":         r.Header,
```

Comment:
Is it worth logging headers before we add headers? Maybe we should move this log to the end. We should also probably start only logging this if requester != "" cause it's been noisy.

Reply:
This is logging the request headers that we are receiving not the ones we set. 

Sure, yeah we can add that conditional.